Libplanet.Tools/Mpt.cs

```diff
@@ -0,0 +1,88 @@
+#nullable enable
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using Cocona;
+using Cocona.Help;
+using Libplanet.RocksDBStore;
+using Libplanet.Store.Trie;
+
+namespace Libplanet.Tools
+{
+    public class Mpt
+    {
+        private readonly IImmutableDictionary<string, Func<string, IKeyValueStore>>
+            _kvStoreConstructors =
+                ImmutableSortedDictionary<string, Func<string, IKeyValueStore>>.Empty
+                    .Add("default", kvStorePath => new DefaultKeyValueStore(kvStorePath))
+                    .Add("rocksdb", kvStorePath => new RocksDBKeyValueStore(kvStorePath));
```

Comment:
How about using a dictionary literal and immediately calling its [`.ToImmutableSortedDictionary()`][1] method?  That would be more readable.

[1]: https://docs.microsoft.com/en-us/dotnet/api/system.collections.immutable.immutablesorteddictionary.toimmutablesorteddictionary